src/bio/bio_xstream.c

```diff
@@ -125,6 +127,19 @@ bio_spdk_env_init(void)
 			D_ERROR("Failed to read acceleration properties, "DF_RC"\n", DP_RC(rc));
 			goto out;
 		}
+
+		/**
+		 * Read flag to indicate whether to enable the SPDK JSON-RPC server and the
+		 * socket file address from the JSON config used to initialize SPDK subsystems.
+		 */
+		rc = bio_read_rpc_srv_settings(nvme_glb.bd_nvme_conf, &enable_rpc_srv,
+					       &nvme_glb.bd_rpc_srv_addr);
+		if (rc != 0) {
+			D_ERROR("Failed to read SPDK JSON-RPC server settings, "DF_RC"\n",
+				DP_RC(rc));
+			goto out;
+		}
+		nvme_glb.bd_enable_rpc_srv = enable_rpc_srv;
```

Comment:
We should probably check to see if this is a release build and disallow starting the JSON-RPC server if so, right? I think it would be relative simple, e.g.:
```suggestion
		nvme_glb.bd_enable_rpc_srv = enable_rpc_srv;
		#ifdef DAOS_RELEASE_BUILD
		if (enable_rpc_srv) {
			D_ERROR("JSON-RPC server may not be enabled for release builds.\n");
			goto out;
		}
		#endif
```

Reply:
done